srv/salt/ceph/updates/salt/default.sls

```diff
@@ -3,9 +3,10 @@ update salt:
     - pkgs:
       - salt-minion 
     - dist-upgrade: True
+    - failhard: True
 
 restart salt-minion:
-  module.run:
-    - name: service.restart
-    - m_name: salt-minion
-
+  cmd.run:
+    - name: "systemctl restart salt-minion"
```

Comment:
I'm probably being irritatingly nit-picky here, but would you be OK with the following variant, to match what's in https://docs.saltproject.io/en/latest/faq.html#what-is-the-best-way-to-restart-a-salt-minion-daemon-using-salt-after-upgrade?

```suggestion
    - name: "salt-call service.restart salt-minion"
```